tests/conftest.py

```diff
@@ -1154,9 +1154,16 @@ def cluster_load(
     io_in_bg = config.RUN.get('io_in_bg')
     log_utilization = config.RUN.get('log_utilization')
     io_load = config.RUN.get('io_load')
+    cluster_load_error = None
+    cluster_load_error_msg = (
+        "Cluster load might not work correctly during this run, because "
+        "it failed with an exception: %s"
```

Comment:
variable is missing - `%s`

Reply:
it's not missing, it's formatted later on in logger itself:  https://github.com/red-hat-storage/ocs-ci/pull/3192/files#diff-e52e4ddd58b7ef887ab03c04116e676f6280b824ab7469d5d3080e5cba4f2128R1185